.eslintrc.js

```diff
@@ -29,11 +42,23 @@ module.exports = {
     'WeakMap': true,
   },
   rules: {
-    'require-yuidoc-access': 'error',
-    'no-const-outside-module-scope': 'error',
+    'ember-internal/require-yuidoc-access': 'error',
+    'ember-internal/no-const-outside-module-scope': 'error',
 
     // temporarily disabled
     'no-unused-vars': 'off',
     'comma-dangle': 'off',
   },
 };
+
+if (isEmberJSBuildLinked()) {
+  delete options.plugins;
+
+  options.rules['require-yuidoc-access'] = options.rules['ember-internal/require-yuidoc-access'];
```

Comment:
@rwjblue can we put that in a loop that just does that for all rules starting with `ember-internal`?